### PR TITLE
Bug 1873175: Allow CSI Drivers to run on master nodes

### PR DESCRIPTION
**Pull request by huffmanca**
Updates the CSI Driver deployment files to allow scheduling on master nodes.

### CODE REVIEW

File: assets/csidriveroperators/ovirt/07_deployment.yaml [hunk 1]
```diff
@@ -77,9 +77,14 @@ spec:
...
               cpu: 10m
+          nodeSelector:
+            node-role.kubernetes.io/master: ""
           tolerations:
             - key: CriticalAddonsOnly
               operator: Exists
+            - key: node-role.kubernetes.io/master
+              operator: Exists
+              effect: "NoSchedule"
           args:
```
Comment (gnufied) [inline assets/csidriveroperators/ovirt/07_deployment.yaml:88]: this was a problem in original template too. The toleration and selector belong to pod's template not to container.

Comment (huffmanca) [reply assets/csidriveroperators/ovirt/07_deployment.yaml:87]: I believe this has been updated . I went ahead and moved the other toleration, as I saw you had done this in your PR to the ovirt operator itself.

**Review by bertinatto — COMMENTED** [review]
I built and ran the CSO from this branch and it correctly scheduled the AWS EBS CSI driver in a control plane node.

CSO itself already has the node selector + toleration to run in the control plane.

/lgtm

**Pull request merged**